dapp/src/components/buySell/SwapCurrencyPill.js

```diff
@@ -364,6 +364,21 @@ const SwapCurrencyPill = ({
         (selectedSwap.amountReceived * priceToleranceValue) / 100
       : null
 
+  const pv =
```

Comment:
What does `pv` stand for? Can you replace with a more descriptive variable please?